OMCompiler/Makefile.omdev.mingw

```diff
@@ -501,7 +499,7 @@ runtimeCPPmsvcinstall: getMSVCversion
 	# Force make to do suitesparse and sundials in the correct order and wait for umfpack to be finished before starting sundials
 	# This is a terrible idea
 	rm -rf $(OMBUILDDIR)/lib/omc/msvc/umfpack.lib
-	$(MAKE) -f $(defaultMakefileTarget) OMBUILDDIR=$(OMBUILDDIR) Umfpack_msvc
+	$(MAKE) -f $(defaultMakefileTarget) OMBUILDDIR=$(OMBUILDDIR) suitesparse_msvc
```

Comment:
I forgot the discussion last time but was anyone still using the msvc targets? If not they should be removed. Otherwise it is just extra work every time.

Reply:
I think the Cpp guys are using it and they say it is significantly faster on Windows. But I'm not so sure now that I remember that correctly.

Reply:
Same. I think at some point in the meetings no one replied to the question who is using these exact targets.